experimental/dds/stack/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@fluid-experimental/stack",
+  "version": "2.0.0-internal.1.2.0",
+  "description": "Distributed stack",
+  "homepage": "https://fluidframework.com",
```

Comment:
You can use`"private: true"` if you want to use this internally without publishing, and/or if you want to iterate on it a bit more before asking bold customers to try it out.

Reply:
Oh yes! That is definitely what I want to do (see recent comment). Thanks.